jsk_robot_common/jsk_robot_startup/lifelog/common_logger.launch

```diff
@@ -295,15 +333,48 @@
     </node>
 
     <!-- object detection logger -->
-    <node if="$(arg save_object_detection)"
-          name="object_detection_logger"
-          pkg="jsk_robot_startup" type="object_detection_logger.py"
+    <group if="$(arg save_object_detection)">
+      <node if="$(arg save_edgetpu_object_detector)"
+            name="object_detection_logger"
+            pkg="jsk_robot_startup" type="mongo_record.py"
+            machine="$(arg machine)"
+            respawn="$(arg respawn)">
+        <rosparam subst_value="true">
+          database: $(arg database)
+          topics:
+          - /edgetpu_human_pose_estimator/output/class
```

Comment:
https://github.com/jsk-ros-pkg/coral_usb_ros#object-detector-edgetpu_object_detectorpy-1

please add `output/rects`

Reply:
typo: human -> object detector?

Reply:
Thank you very much for reviewing. 
I fixed it in [d19109a](https://github.com/jsk-ros-pkg/jsk_robot/pull/1728/commits/d19109a730519f57e463801d6d86f08e3f30d8a7) and [d034ac9](https://github.com/jsk-ros-pkg/jsk_robot/pull/1728/commits/d034ac969beff94b096e5af46fe8e0c3827e7fac).